runtime/pom.xml

```diff
@@ -142,6 +143,7 @@
             <test>SpecV1LinkingTest.test118</test>
             <test>SpecV1LinkingTest.test121</test>
             <test>SpecV1LinkingTest.test122</test>
+            <test>SpecV1ElemTest.test95</test>
```

Comment:
This test is failing because we need to keep the root instance of exported Global FuncRefs.

There is a little design issue to overcome:
- a `FuncRef` should be able to be associated with an `Instance`
- `FuncRef` is defined in the `wasm` module
- `Instance` is defined in the `runtime` module and is not visible in the `wasm` module

For Tables I initially defined `TableInstance` in the `runtime` module, but this approach is falling short here where initializers etc. are involved.

Happy to hear thoughts! cc. @bhelx @danielperano

Reply:
Resisting hard to break the abstraction layer in between `runtime` and `wasm` modules.
This is a possible implementation that does the job: https://github.com/dylibso/chicory/pull/310/commits/3f5aa57ffffd6ffa4cb77177f1021e2d512878dd